projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -228,106 +239,112 @@
-  "ars_nouveau.glyph_desc.flare": "当对着火的实体使用，爆燃会造成爆发伤害，然后向附近实体蔓延火焰并造成伤害。比伤害构件造成显著更多的伤害。可用延长时效、强化和范围效果增强。",
+  "ars_nouveau.glyph_desc.fell": "收获整颗树、蘑菇、仙人掌和其他植物。强化后可破坏更高硬度的材料。扩散会增加一次能破坏的方块数量。",
+  "ars_nouveau.glyph_desc.firework": "在目标位置或目标实体处发射烟花，强化能增加烟火之星数量，延长时效增加飞行时间。法术发射的烟花会尝试模拟施法者物品栏中的烟花。在法术炮塔中用触摸形态施放则会类似使用发射器发射。",
+  "ars_nouveau.glyph_desc.flare": "当对着火的实体使用，爆燃会造成爆发伤害，然后向附近实体蔓延火焰并造成伤害。此构件比伤害构件造成的伤害更高。可用延长时效、强化和扩散增强。",
   "ars_nouveau.glyph_desc.fortune": "增加用伤害杀死生物、用破坏挖掘方块的掉落率。不能与提取叠加。",
   "ars_nouveau.glyph_desc.freeze": "小范围冻结水体和熔岩，或短时间使目标缓慢。",
-  "ars_nouveau.glyph_desc.glide": "造成滑翔效果，能使被影响者如同穿戴了鞘翅一般滑翔。",
-  "ars_nouveau.glyph_desc.gravity": "使方块或实体掉落。当使用延长时效增强时，被影的响玩家将不能飞行，并获得重力状态。该状态下的实体会急速坠落，并受到双倍跌落伤害。",
+  "ars_nouveau.glyph_desc.glide": "使被影响者如同穿戴了鞘翅一般滑翔。",
+  "ars_nouveau.glyph_desc.gravity": "使方块或实体掉落。当使用延长时效增强时，被影的响玩家将不能飞行，并获得重力效果。该效果下的实体会急速坠落，并受到双倍跌落伤害。",
   "ars_nouveau.glyph_desc.grow": "使植物生长加速，如同被施加骨粉一般。",
-  "ars_nouveau.glyph_desc.gust": "使目标被推离施法者少量距离",
-  "ars_nouveau.glyph_desc.harm": "你的初始法术构件。伤害一个目标。可用强化增加伤害，或使用延长时效造成中毒状态。注意，没有延迟的多重伤害会因为击中后的无敌时间无法生效。",
+  "ars_nouveau.glyph_desc.gust": "使目标被施法者推开少量距离",
+  "ars_nouveau.glyph_desc.harm": "你的初始法术构件。伤害一个目标。可用强化增加伤害，或使用延长时附加中毒效果。注意，没有延迟的多重伤害会因为击中后的无敌时间无法生效。",
   "ars_nouveau.glyph_desc.harvest": "当用于成熟的作物，该法术效果会在不破坏植物的情况下获得完全成熟后的产品。",
-  "ars_nouveau.glyph_desc.heal": "对目标治疗少量生命。对亡灵生物使用时，会造成与治疗量等价的魔法伤害。",
-  "ars_nouveau.glyph_desc.hex": "对目标造成最高5级的脆弱诅咒。当目标在中毒、凋零或燃烧时，脆弱诅咒会少量增加其受到的任何伤害。此外，脆弱诅咒还会将目标魔力和生命再生的速度减少一半。",
+  "ars_nouveau.glyph_desc.heal": "治疗目标少量生命。对亡灵生物使用时，会造成与治疗量等价的魔法伤害。",
+  "ars_nouveau.glyph_desc.hex": "对目标附加最高5级的脆弱诅咒。当目标在中毒、凋零或燃烧时，脆弱诅咒会少量增加其受到的所有伤害。此外，脆弱诅咒还会减半目标魔力和生命恢复的速度。",
   "ars_nouveau.glyph_desc.ignite": "使方块或生物短时间着火。敏感构件能防止点着方块。",
-  "ars_nouveau.glyph_desc.intangible": "使方块暂时变成空气。使用强化法术增强后，便可以处理高挖掘等级方块。无形还能被范围效果、缩短时效和延长时效增强。",
+  "ars_nouveau.glyph_desc.intangible": "使方块暂时变成空气。使用强化法术增强后，便可以处理高挖掘等级方块。虚化还能被扩散、缩短时效和延长时效增强。",
   "ars_nouveau.glyph_desc.interact": "模仿玩家与方块或实体交互。对于触及拉杆、箱子和动物很有用。",
   "ars_nouveau.glyph_desc.invisibility": "使实体短时间隐形。",
-  "ars_nouveau.glyph_desc.launch": "将目标推向空中。可用于大跳或登山！",
+  "ars_nouveau.glyph_desc.launch": "将目标推向空中。非常适合鞘翅起飞或登山！",
   "ars_nouveau.glyph_desc.leap": "将目标向其注视的方向推出。强化可增加移动距离。",
-  "ars_nouveau.glyph_desc.light": "向方块施展会创造一个永久光源。被强化可使提升光源亮度，最高与荧石相同，被抑制则会降低光源亮度。向自己施展会获得夜视。向其他实体施展则会造成夜视和发光。",
-  "ars_nouveau.glyph_desc.lightning": "在目标位置召唤闪电。向被击中的实体造成触电状态。触电状态会使得后续的闪电造成额外伤害，并最高使触电状态的等级上升为III级。闪电也会对潮湿和穿戴RF装备的实体造成额外伤害。可被强化、抑制和延长时效构件增强。",
-  "ars_nouveau.glyph_desc.linger": "创造一个滞留场，短期内持续对附近实体施加法术。与敏感构件配合会将法术目标改为方块。范围效果能增大生效范围，加速构件能增加施法速度，延长时效则能增加滞留场持续时间。",
+  "ars_nouveau.glyph_desc.light": "向方块施展会创造一个永久光源。使用强化或抑制构件可以改变光源亮度，默认和火把一致。向自己施展会获得夜视。向其他实体施展则会附加夜视和发光。",
+  "ars_nouveau.glyph_desc.lightning": "在目标位置召唤闪电，并向被击中的实体附加感电效果。感电效果会增加闪电造成的伤害并上升一级，最高III级。闪电构件对潮湿和穿戴RF装备的实体造成的伤害更高。可被强化、抑制和延长时效构件增强。",

```

不知道我理解的对不对，这样写感觉会清楚一点
```suggestion
  "ars_nouveau.glyph_desc.lightning": "在目标位置召唤闪电，并向被击中的实体附加感电效果。感电效果会增加受到的闪电伤害，并在每次被击中时提高一级，最高III级。闪电构件对潮湿和穿戴RF装备的实体造成的伤害更高。可被强化、抑制和延长时效构件增强。",
```